src/tools/dashboard/src/reference.rs

```diff
@@ -0,0 +1,287 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! Utilities to extract examples from The Rust Reference, run them through RMC,
+//! and display their results.
+
+use pulldown_cmark::{Parser, Tag};
+use std::io::{BufRead, BufReader};
+use std::process::Stdio;
+use std::{
+    collections::HashMap,
+    path::{Path, PathBuf},
+    process::Command,
+};
```

Comment:
Why are you doing this for some `std` imports and not others?

Reply:
I trusted `rustfmt` and let it take care of that. Cleaned up.